shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/map/RouteLayerGeneratorTest.kt

```diff
@@ -19,14 +19,14 @@ class RouteLayerGeneratorTest {
 
         assertEquals(
             listOf(
-                "route-layer-Red",
                 "route-layer-Orange",
-                "route-layer-Red-alerting-bg",
-                "route-layer-Red-shuttled",
-                "route-layer-Red-suspended",
+                "route-layer-Red",
                 "route-layer-Orange-alerting-bg",
```

Comment:
question(non-blocking): Is this just b/c of sort order changing for the test data?

Reply:
Yes, the sort order had previously been undefined and is now defined.